src/java.base/share/classes/java/lang/classfile/package-info.java

```diff
@@ -242,25 +241,49 @@
  * <p>
  * If a constant pool entry has a nominal representation then it provides a
  * method returning the corresponding nominal descriptor type e.g.
- * method {@link jdk.internal.classfile.constantpool.ClassEntry#asSymbol} returns
+ * method {@link java.lang.classfile.constantpool.ClassEntry#asSymbol} returns
  * {@code ClassDesc}.
  * <p>
  * Where appropriate builders provide two methods for building an element with
  * symbolic information, one accepting nominal descriptors, and the other
  * accepting constant pool entries.
  *
+ * <h3>Consistency checks, syntax checks and verification</h3>
+ * No consistency checks are performed while building or transforming classfiles
+ * (except for null arguments checks). All builders and classfile elements factory
+ * methods accepts provided information without implicit validation.
+ * However fatal inconsistencies (like for example invalid code sequence or
+ * unresolved labels) affects internal tools and may cause exceptions later in
+ * the classfile building process.
+ * <p>
+ * Basic syntax control can be achieved using symbolic descriptors. For example
```

Comment:
What does "syntax control" mean? The concept is used, but not defined, in the javadoc.

Reply:
I've reformulated and extended the paragraph a bit and added another snippet.